src/anyvlm/restapi/vlm.py

```diff
@@ -9,3 +24,45 @@ def ingest_vcf(vcf_path: Path) -> None:
     :param vcf_path: VCF file location
     """
     raise NotImplementedError
+
+
+@app.get(
+    "/vlm-query",
+    summary="Provides counts of occurrences of a single sequence variant, broken down by zygosity",
+    description="Search for a single sequence variant and receive a count of its observed occurrences broken down by zygosity, in accordance with the Variant-Level Matching protocol",
```

Comment:
another suggestion feel free to take or leave

```suggestion
    description="Search for a single sequence variant and receive allele counts by zygosity, in accordance with the Variant-Level Matching protocol",
```